lib/cisco_node_utils/fabricpath_topology.rb

```diff
@@ -72,54 +67,47 @@ def fabricpath_feature_set(fabricpath_set)
       FabricpathGlobal.fabricpath_feature_set(fabricpath_set)
     end
 
-    def state
-      result = config_get('fabricpath_topology', 'state', @topo_id)
-      return default_state if result.nil?
-      case result
-      when /Up/
-        return 'up'
-      when /Down/
-        return 'default'
+    def member_vlans
+      config_get('fabricpath_topology', 'member_vlans',
+                 @topo_id).gsub(/\s+/, '')
```

Comment:
Can you add a small comment to this method showing sample output from config_get? It's not immediately obvious why the gsub is needed.